### PR TITLE
Agent release 3.18.4 changelog

### PR DESCRIPTION
**Pull request by jrobsonchase**

### CODE REVIEW

**Comment by dthomasngrokker** [comment]
I think you also need to update the Latest version in the version-support-policy file

File: docs/agent/version-support-policy.mdx [hunk 1]
```diff
@@ -44,10 +44,7 @@ Extended support beyond LTS is available to customers on a commercial basis. It
 
 | Support Status | Release | Release Date | Latest | Active Support | End of Life |
 | -------------- | ------- | ------------ | ------ | -------------- | ----------- |
-| Active         | 3.18x   | 2024-11-07   | 3.18.3 | 2025-11-07     | 2026-01-07  |
-| Active         | 3.18x   | 2024-10-31   | 3.18.2 | 2025-10-31     | 2025-12-10  |
-| Active         | 3.18x   | 2024-10-17   | 3.18.1 | 2025-10-24     | 2025-12-10  |
-| Active         | 3.18x   | 2024-10-17   | 3.18.0 | 2025-10-17     | 2025-12-10  |
```
Comment (dthomasngrokker) [inline docs/agent/version-support-policy.mdx:50]: These should not be removed

**Review by dthomasngrokker — CHANGES_REQUESTED** [review]

**Review by dthomasngrokker — APPROVED** [review]

File: docs/agent/version-support-policy.mdx (hunk 1 above)
Comment (jrobsonchase) [reply docs/agent/version-support-policy.mdx:50]: Disagree - the support policy is for 3.18.x, not a separate support policy for each patch version.

Comment (dthomasngrokker) [reply docs/agent/version-support-policy.mdx:50]: You right

**Pull request merged**